sparrow-py/pytests/conftest.py

```diff
@@ -2,8 +2,66 @@
 import pytest
 from sparrow_py import init_session
 
+from typing import Any
+from typing import Callable
+from typing import Literal
+from typing import Union
+import pandas as pd
+import pytest
+import functools
+import os
+import sparrow_py
+
 
 @pytest.fixture(autouse=True, scope="session")
 def session() -> None:
     """Automatically initialize the session for this PyTest session."""
     init_session()
+
+
+def pytest_addoption(parser: pytest.Parser):
+    parser.addoption("--save-golden", action="store_true", help="update golden files")
+
+@pytest.fixture
+def golden(request: pytest.FixtureRequest, pytestconfig: pytest.Config):
+    output = 1
```

Comment:
I use the index so we can have multiple golden checks in a test.

I could omit the `_1` if it is the first iteration, which would simplify some of the output files.